google/cloud/security/inventory/inventory_loader.py

```diff
@@ -256,36 +247,47 @@ def main(argv):
     # Otherwise, there is a gap where the ratelimiter from one pipeline
     # is not used for the next pipeline using the same API. This could
     # lead to unnecessary quota errors.
-    crm_rate_limiter = CloudResourceManagerClient.get_rate_limiter()
-    admin_directory_rate_limiter = AdminDirectoryClient.get_rate_limiter()
+    # TODO: Provide the api client from the api client module instead of here.
+    max_crm_calls = configs.get('max_crm_api_calls_per_100_seconds', 400)
+    crm_rate_limiter = RateLimiter(max_crm_calls, 100)
+    crm_api_client = crm.CloudResourceManagerClient(
```

Comment:
nit: I thought we had decided to push rate limit definitions/creation into the modules that support that code.